components/bpmn/bpmn-rest/src/main/java/org/wso2/carbon/bpmn/rest/service/stats/UserService.java

```diff
@@ -113,32 +126,39 @@ public ResponseHolder getAvgDurationForTasksCompletedByUser() throws UserStoreEx
         List listOfUsers = new ArrayList<>();
         ResponseHolder response = new ResponseHolder();
         try{
-        String[] users = (String[]) getUserList().getData().toArray();
-        for (String u : users) {
-
-            UserTaskDuration userInfo = new UserTaskDuration();
-            userInfo.setUserName(u);
-
-            long count = BPMNOSGIService.getHistoryService()
-                    .createHistoricTaskInstanceQuery().taskAssignee(u).finished().count();
-            if (count == 0) {
-                double avgTime = 0;
-                userInfo.setAvgTimeDuration(avgTime);
-            } else {
-                List<HistoricTaskInstance> taskList = BPMNOSGIService.getHistoryService()
-                        .createHistoricTaskInstanceQuery().taskAssignee(u).finished().list();
-                double totalTime = 0;
-                double avgTime = 0;
-                for (HistoricTaskInstance instance : taskList) {
-                    double taskDuration = instance.getDurationInMillis();
-                    totalTime = totalTime + taskDuration;
+            String[] users = (String[]) getUserList().getData().toArray();
+            for (String u : users) {
+
+                UserTaskDuration userInfo = new UserTaskDuration();
+                userInfo.setUserName(u);
+
+                String assignee ;
+                if ( tenantDomain == DOMAIN_OF_SUPER_TENANT ) {
```

Comment:
The string comparison is wrong. Is this intentional? seems to be a bug in the code